documentation/docs/install-pmm/install-pmm-client/connect-database/mongodb.md

```diff
@@ -1,50 +1,56 @@
-- You have `adminUserAnyDatabase` or superuser role privilege to any database servers that you want to monitor.
-- Your MongoDB server is version 4.0 or higher.
+- [PMM Server installed](../../install-pmm-server/index.md) and running with a known IP address or hostname accessible from the Client node.
+- [PMM Client installed](../../install-pmm-client/index.md) and the [nodes are registered with PMM Server](../../register-client-node/index.md).
+- Superuser (root) access on the Client host.
+- `adminUserAnyDatabase` or superuser role privilege on the MongoDB database servers that you want to monitor.
+- MongoDB server version 4.0 or higher.
 
 ## Create PMM account and set permissions
 
-We recommend using a dedicated account to connect PMM Client to the monitored database instance.
+We recommend using a dedicated account to connect PMM Client to the monitored database instance. The permissions required depend on which PMM features you plan to use.
 
-Run the example codes below in a `mongo` session to:
+Run the example commands below in a mongo shell session to:
 
--  create custom roles with the privileges required for creating/restoring backups and working with Query Analytics (QAN)
--  create/update a database user with these roles above, plus the built-in  `clusterMonitor` role
+-  Create custom roles with the privileges required for creating/restoring backups and working with Query Analytics (QAN).
+-  Create/update a database user with these roles, plus the built-in  `clusterMonitor` role.
   
-!!! caution alert alert-warning ""
+!!! caution alert alert-warning "Important"
     Values for username (`user`) and password (`pwd`) are examples. Replace them before using these code snippets.
 
-=== "Create a role with monitoring and QAN privileges"
-    This role grants the essential minimum privileges needed for monitoring and QAN. 
+=== "Monitoring and QAN privileges"
+    This role grants the essential minimum privileges needed for monitoring and QAN:
+
         ```{.javascript data-prompt=">"}
         db.getSiblingDB("admin").createRole({
         "role": "explainRole",
         "privileges": [
             {
                 "resource": { "db": "", "collection": "" },
-                "actions": [ "dbHash", "find", "listIndexes", "listCollections"  ]
+                "actions": [ "dbHash", "find", "listIndexes", "listCollections", "collStats", "dbStats", "indexStats" ]
             },
             {
-                "resource": { "db": "", "collection": "system.version"  },
+                "resource": { "db": "", "collection": "system.version" },
                 "actions": [ "find" ]
-            }
+            },
+            {
+                "resource": { "db": "", "collection": "system.profile" },
+                "actions": [ "dbStats", "collStats", "indexStats" ]
+            }         
         ],
-        "roles": []
+        "roles": [ { role: "directShardOperations", db: "admin" } ]
```

Comment:
does this role exist in older MongoDB versions? If it doesn' will this command fail on them?

Reply:
it doesn't. This is why it is listed only in the tab pertaining to Mongodb 8.0